apps/examples/ble_rmc/ble_rmc_main.c

```diff
@@ -67,27 +68,40 @@ static void ble_device_scanned_cb(ble_scanned_device *scanned_device)
 	return;
 }
 
-static void ble_device_disconnected_cb(ble_conn_handle conn_id)
+static void ble_device_disconnected_cb(ble_client_ctx *ctx)
 {
-	RMC_LOG(RMC_TAG, "'%s' is called\n", __FUNCTION__);
+	RMC_LOG(RMC_TAG, "'%s' is called[%p]\n", __FUNCTION__, ctx);
 	return;
 }
 
-static void ble_device_connected_cb(ble_device_connected *dev)
+static void ble_device_connected_cb(ble_client_ctx *ctx, ble_device_connected *dev)
 {
 	int ret;
-	RMC_LOG(RMC_TAG, "'%s' is called\n", __FUNCTION__);
-	ret = sem_post(&g_conn_sem);
+	RMC_LOG(RMC_TAG, "'%s' is called[%p]\n", __FUNCTION__, ctx);
 
 	g_conn = dev->conn_handle;
-	RMC_LOG(RMC_TAG, "Conn Handle : %d\n", g_conn);
+	RMC_LOG(RMC_TAG, "Conn Handle : %d\n", dev->conn_handle);
+	RMC_LOG(RMC_TAG, "Conn Bonded : %d\n", dev->is_bonded);
+	RMC_LOG(RMC_TAG, "Conn CI : %d\n", dev->conn_info.conn_interval);
+	RMC_LOG(RMC_TAG, "Conn SL : %d\n", dev->conn_info.slave_latency);
+	RMC_LOG(RMC_TAG, "Conn MTU : %d\n", dev->conn_info.mtu);
+	RMC_LOG(RMC_TAG, "Conn MAC : %02x:%02x:%02x:%02x:%02x:%02x\n", 
+		dev->conn_info.addr.mac[0],
+		dev->conn_info.addr.mac[1],
+		dev->conn_info.addr.mac[2],
+		dev->conn_info.addr.mac[3],
+		dev->conn_info.addr.mac[4],
+		dev->conn_info.addr.mac[5]
+	);
+
+	ret = sem_post(&g_conn_sem);
```

Comment:
The return of `sem_post` is not used anywhere.
So I think it is not needed or it is better to check the return.

Reply:
I fixed it